README.md

```diff
@@ -1,22 +1,22 @@
 # kubevirt-apb
 This *Ansible Playbook Bundle* deploys [Kubevirt](http://www.kubevirt.io)
+
 As the resources deployed need cluster-admin privileges, credentials of a super user are required for the APB to execute correctly
 
 # Parameters used
 
-- tag Defaults to 0.2.0
-- admin_user
-- admin_password
-- release whether to download the templates used from kubevirt github releases or render them from local templates
-- openshift . whether to execute additional scc policy commands. Defaults to true
+- *tag*  Tag to use for kubevirt related images (defaults to 0.2.0)
+- *admin_user*
+- *admin_password*
+- *release* Whether to download the templates used from kubevirt github releases or render them from local templates
 
 ## How to use:
 
 #### Deploy from the UI
```

Comment:
I know that this PR doesn't touch this part, but I think you can just fix this typo here.
I think it should be `Deploy from the CLI